services/alert/service.go

```diff
@@ -176,30 +182,25 @@ func NewService(d Diagnostic, disabled map[string]struct{}, topicBufLen int) *Se
 const (
 	// Public name of the handler specs store.
 	handlerSpecsAPIName = "handler-specs"
-	// Public name of the handler specs store.
-	topicStatesAPIName = "topic-states"
 	// The storage namespace for all task data.
-	alertNamespace = "alert_store"
+	AlertNameSpace = "alert_store"
```

Comment:
More comments explaining what is used for V1 and what is used for V2 here. -- @dgnorton